### ISSUE TITLE
Measure performance using test set instead of just making predictions on test set

### ISSUE THREAD

**Issue by lgragert** [lgragert/nn-sero-pytorch#29]
Last time all alleles with serology labels were either in training or validation.

Test set was only unlabeled data.

We could have labeled data in the test set to measure performance this time. We don't have to move every case either to training/validation. 

This is a better study design to measure NN performance. The reason for that is validation set influences training of the model.
